Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1.5
```

Comment:
**nitpick**: could you state the expected usage of images built with these dockerfiles? Something like

```suggestion
# syntax=docker/dockerfile:1.5
# Base image for running Orquestra tasks with custom environment.
```